lib/api/tenure/v1/types.ts

```diff
@@ -65,36 +70,49 @@ export interface LegacyReference {
   name: string;
   value: string;
 }
-
+export interface FurtherAccountInformation {
+  isRentAccountRequired: boolean | null;
+  noRentAccountReason: string | null;
+  rentLetterSentDate: string | null;
+  rentCardGivenDate: string | null;
+  tenureAcceptedDate: string | null;
```

Comment:
Shouldn't this be a date time date type?

Reply:
Thanks for the flag on this. TLDR updated to Date. Kinda gets tricky because the main Tenure type uses some string types where it could/should be date. Had a chat and thought was good to contribute it as it should be - and this one will only impact TA directly.